components/server/src/server.ts

```diff
@@ -42,19 +40,19 @@ import { increaseHttpRequestCounter, observeHttpRequestDuration } from './promet
 import { OAuthController } from './oauth-server/oauth-controller';
 import { HeadlessLogController } from './workspace/headless-log-controller';
 import { NewsletterSubscriptionController } from './user/newsletter-subscription-controller';
-import { Config } from './config';
+import { Config, ConfigEnv } from './config';
+import { Env } from './env';
 
 @injectable()
 export class Server<C extends GitpodClient, S extends GitpodServer> {
     static readonly EVENT_ON_START = 'start';
 
-    @inject(Env) protected readonly env: Env;
     @inject(Config) protected readonly config: Config;
+    @inject(Env) protected readonly env: Env;
     @inject(SessionHandlerProvider) protected sessionHandlerProvider: SessionHandlerProvider;
     @inject(Authenticator) protected authenticator: Authenticator;
     @inject(UserController) protected readonly userController: UserController;
     @inject(EnforcementController) protected readonly enforcementController: EnforcementController;
```

Comment:
Why did you remove TheiaPluginController in this PR?

Reply:
We don't use it anymore.